x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/GetDataFrameAnalyticsAction.java

```diff
@@ -34,6 +34,8 @@ public Response newResponse() {
 
     public static class Request extends AbstractGetResourcesRequest {
 
+        public static final ParseField ALLOW_NO_MATCH = new ParseField("allow_no_match");
```

Comment:
Is this field ever used?

Reply:
It is used from `MlRequestConverters.getDataFrameAnalytics`